tests/Directory.Build.props

```diff
@@ -0,0 +1,16 @@
+<Project>
+  <Import Project="..\Directory.Build.props" />
+
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <UseVSTestRunner>true</UseVSTestRunner>
```

Comment:
I believe this should be set in Versions.props instead: https://github.com/mono/linker/blob/e2ce67a42fac9915e764b52fcd15bbbb22576711/eng/Versions.props#L29

Reply:
What is the difference?

Reply:
Just a note that Arcade expects these feature switches to be set in Versions.props.

Reply:
I see, does it set versions based on that?